lib/server-builder.js

```diff
@@ -72,7 +72,7 @@ class ServerBuilder {
   }
 
   getCommand () {
-    const cmd = system.isWindows() ? 'gradlew.bat' : './gradlew';
+    const cmd = path.resolve(this.serverPath, system.isWindows() ? 'gradlew.bat' : 'gradlew');
```

Comment:
I'm not quite sure this is going to work as expected, at least on Windows where .bat files are not executable on their own.
I assume it would be safer to run these scripts using `cmd.exe /c` on windows and using `bash` on other systems

Reply:
oh, i see. > win

Yes, bash might be safe in case users change the permission by some reasons...
Let me keep .bat for Win in this PR.

I'll create another pr to add bash and cmd.exe to separate versions.

Reply:
The below full path worked in both powershell and command prompt on Win 10, but `cmd.exe /c` did not in the command prompt. (Access denied error occurred). Powershell worked with the `/c`.

So, on Win, it seems like only with `gradlew.bat` or the full path to `gradlew.bat` is safer.

```
C:\Users\kazu\Downloads\appium-espresso-driver-master\appium-espresso-driver-master\espresso-server>C:\kazu\kazu\Downloads\appium-espresso-driver-master\appium-espresso-driver-master\espresso-server\gradlew.bat

FAILURE: Build failed with an exception.

* Where:
Build file 'C:\Users\kazu\Downloads\appium-espresso-driver-master\appium-espresso-driver-master\espresso-server\app\build.gradle.kts' line: 1

* What went wrong:
An exception occurred applying plugin request [id: 'com.android.application']
> Failed to apply plugin 'com.android.internal.application'.
   > Android Gradle plugin requires Java 11 to run. You are currently using Java 1.8.
     You can try some of the following options:
       - changing the IDE settings.
       - changing the JAVA_HOME environment variable.
       - changing `org.gradle.java.home` in `gradle.properties`.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 1s
<-------------> 0% WAITING
> IDLE
```